pixelpact/src/helpers.js

```diff
@@ -0,0 +1,4 @@
+export function getLocalAddress(fastifyInstance) {
+  let address = fastifyInstance.addresses().find((e) => e.family === "IPv4");
```

Comment:
```suggestion
  const address = fastifyInstance.addresses().find((e) => e.family === "IPv4");
```

Reply:
`address` can potentially be `undefined`. I think we need to check for the `undefined` case.

Reply:
Will throw here an explicit exception if `address` is undefined